tests/foreman/cli/test_host.py

```diff
@@ -1080,14 +1080,14 @@ def test_positive_parameter_crud(self):
 
         :CaseImportance: Critical
         """
-        name = valid_data_list()[0].lower()
-        value = valid_data_list()[0]
+        name = next(iter(valid_data_list())).lower()
+        value = valid_data_list()[name]
```

Comment:
I would drop `lower()` here.
You take iterator of dict keys, take first value, call `lower()` on that and then use this value on the same dict to access value under obtained key. This works only because dict keys, apparently, are already all lowercase.

At best, this `lower()` does nothing. At worst, it will cause failure when there is none.

I realize it's been here for a while already. That's why I don't consider this blocking.

Reply:
I drop it :)